### DISCUSSION TITLE
Librephotos

### DISCUSSION THREAD

**Discussion by meichthys** [tteck/Proxmox#345] [Ideas]
https://docs.librephotos.com/1/manual_debian_install/

**Comment by nacho-domatix** [comment]
+1 to this.

Seems to be a nice alternative to Photoprism, which I'm still trying. Seems easy to install but it failed on a clean Ubuntu CT to me.

**Reply by nacho-domatix** [reply]
I tried already Photoprism, I found it's not really opensource, as you have to pay a subscription even for so many features, even basic ones as see maps. And "[PhotoPrism® needs your support!](https://photoprism.app/membership)" is everywhere, can be even annoying in the mobile view and spoil the user experience.

Also I dislike the way to manage folders and files, I mounted a read-only disk, cause I want to make sure it doesn't do anything weird to my files, and it was unable to work that way.

Not gonna be my solution indeed. From what I saw, [Librephotos](https://demo2.librephotos.com/) and [Photoview](https://photos.qpqp.dk/timeline) are the winners, [Lychee](https://lycheeorg.github.io/demo/#) seems to be lack of options and [Pigallery](https://pigallery2.herokuapp.com/gallery/) doesn't have map (yet?).

**Reply by meichthys** [reply]
This may help out: https://github.com/meichthys/foss_photo_libraries

**Reply by nacho-domatix** [reply]
I was using this one: https://github.com/awesome-selfhosted/awesome-selfhosted#photo-and-video-galleries

**Reply by meichthys** [reply]
The repo i linked to is based on that list. It just clarifies the feature set of each project and removes any that are self-hosted but not Free and Open Source.

**Reply by nacho-domatix** [reply]
I found it in [reddit](https://www.reddit.com/r/selfhosted/). It's good to know which products are free and community driven cause otherwise they often end up charging for extra features or with an "Enterpise edition".

**Reply by meichthys** [reply]
> I found it in [reddit](https://www.reddit.com/r/selfhosted/). It's good to know which products are free and community driven cause otherwise they often end up charging for extra features or with an "Enterpise edition".

Exactly. They have their right to charge for pro features but I like to use and contribute to completely open and free projects. That's why I added a "freeness" scale to my list

**Reply by nacho-domatix** [reply]
Free and libre as we say in Spanish :P

**Comment by vladimirperovic** [comment]
+1 for librephotos,
I tried photoprism but upload and processing is very slow
All the best
